### PR TITLE
Update Korean

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

**Comment by marocchino** [comment]
@yous review please.

File: languages/ko.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   "footer-about-host": "워크숍 개최하기",
   "index-header-top": "터미널에서 실행하는 노드(Node)로 <br>공부하실 수 있는 오픈소스 워크숍입니다. <br>스스로 공부하시거나 가까운 워크숍을 방문해 보세요.",
   "index-header-bottom": "<a class=\"big-link\" href=\"#workshopper-list\">기본적인 워크숍</a>을 설치해 바로 시작하거나,<br>무료 <a class=\"big-link\" href=\"https://tinyletter.com/nodeschool\" target=\"_blank\">이메일 뉴스레터</a>를 구독하세요.",
+  "index-chapter-list-loading": "로딩...",
```
Comment (yous) [inline languages/ko.json]: How about '로드 중'?

File: languages/ko.json [hunk 2]
```diff
@@ -90,9 +91,12 @@
   "workshopper-learnyoucouchdb": "웹만으로도 제어 가능한 데이터베이스인 CouchDB에 대해 배워봅시다.",
   "workshopper-learnuv": "즐겁고 유익한 uv를 배우세요. Node.js를 구성하는 라이브러리를 자습할 수 있는 워크숍입니다.",
   "workshopper-learn-generators": "ECMAScript6 Generator를 소개합니다.",
-  "workshopper-tower-of-babel": "연습문제를 통해 ES6의 기능을 소개합니다.",
+  "workshopper-tower-of-babel": "연습 문제를 통해 ES6의 기능을 소개합니다.",
   "workshopper-learnyoureact": "React.js와 서버 사이드 렌더링을 배웁시다!",
   "workshopper-perfschool": "웹 성능 최적화의 미로에서 길을 찾아보세요!",
+  "workshopper-scope-chains-closures": "스코프, 스코프 체인, 클로저, 가비지 콜랙션을 자세히 배우세요.",
```
Comment (yous) [inline languages/ko.json]: 콜랙션 → 컬렉션

**Comment by yous** [comment]
Sorry, but where is the original English version?

**Comment by marocchino** [comment]
In this repo, html is original.
but, You can ref [this](https://gist.github.com/marocchino/55d6306fdc3199e56ee8)

File: languages/ko.json [hunk 3]
```diff
@@ -90,9 +91,12 @@
   "workshopper-learnyoucouchdb": "웹만으로도 제어 가능한 데이터베이스인 CouchDB에 대해 배워봅시다.",
   "workshopper-learnuv": "즐겁고 유익한 uv를 배우세요. Node.js를 구성하는 라이브러리를 자습할 수 있는 워크숍입니다.",
   "workshopper-learn-generators": "ECMAScript6 Generator를 소개합니다.",
-  "workshopper-tower-of-babel": "연습문제를 통해 ES6의 기능을 소개합니다.",
+  "workshopper-tower-of-babel": "연습 문제를 통해 ES6의 기능을 소개합니다.",
   "workshopper-learnyoureact": "React.js와 서버 사이드 렌더링을 배웁시다!",
   "workshopper-perfschool": "웹 성능 최적화의 미로에서 길을 찾아보세요!",
+  "workshopper-scope-chains-closures": "스코프, 스코프 체인, 클로저, 가비지 컬렉션을 자세히 배우세요.",
+  "workshopper-esnext-generation": "ES6의 제너레이터와 그 사용법 제너레이터와 관계에 대해 소개합니다.",
```
Comment (yous) [inline languages/ko.json]: ES6의 이터레이터와 그 사용법, 제너레이터와의 관계에 대해 소개합니다.

**Comment by yous** [comment]
Thanks, I'm done.

**Comment by marocchino** [comment]
Thanks!

File: languages/ko.json [hunk 4]
```diff
@@ -90,9 +91,12 @@
   "workshopper-learnyoucouchdb": "웹만으로도 제어 가능한 데이터베이스인 CouchDB에 대해 배워봅시다.",
   "workshopper-learnuv": "즐겁고 유익한 uv를 배우세요. Node.js를 구성하는 라이브러리를 자습할 수 있는 워크숍입니다.",
   "workshopper-learn-generators": "ECMAScript6 Generator를 소개합니다.",
-  "workshopper-tower-of-babel": "연습문제를 통해 ES6의 기능을 소개합니다.",
+  "workshopper-tower-of-babel": "연습 문제를 통해 ES6의 기능을 소개합니다.",
   "workshopper-learnyoureact": "React.js와 서버 사이드 렌더링을 배웁시다!",
   "workshopper-perfschool": "웹 성능 최적화의 미로에서 길을 찾아보세요!",
+  "workshopper-scope-chains-closures": "스코프, 스코프 체인, 클로저, 가비지 컬렉션을 자세히 배우세요.",
+  "workshopper-esnext-generation": "ES6의 이터레이터와 그 사용법, 제너레이터와의 관계에 대해 소개합니다.",
+  "workshopper-web-audio-school": "음악에 관련된 대화형 수업으로 웹 오디오 API을 배우세요.",
```
Comment (yous) [inline languages/ko.json]: I'm sorry for late, but 'API을' should be 'API를'.

Comment (marocchino) [inline languages/ko.json]: ;_;

Comment (marocchino) [inline languages/ko.json]: I will fix it next update.(I will be next week)

Comment (yous) [inline languages/ko.json]: Okay.

**Pull request merged**